browser/ui/browser_commands.cc

```diff
@@ -1182,4 +1184,80 @@ void SwapTabsInTile(Browser* browser) {
                            /*select_after_move*/ false);
 }
 
+bool CanOpenNewSplitTabsWithSideBySide(Browser* browser) {
+  auto* tab_strip_model = browser->tab_strip_model();
```

Comment:
Should we CHECK() if the side-by-side feature flag is on? other places ditto - though the callsites are checking it but I think having checks in side functions is needed as they are public functions, which anyone could call it without checking it by mistake.